frontend/src/components/preview/PreviewLongTexts.vue

```diff
@@ -0,0 +1,98 @@
+<script setup lang="ts">
+import TextEditor from "@/components/input/TextEditor.vue"
+import PreviewCategory from "@/components/preview/PreviewCategory.vue"
+import PreviewContent from "@/components/preview/PreviewContent.vue"
+import PreviewRow from "@/components/preview/PreviewRow.vue"
+import { LongTexts } from "@/domain/documentUnit"
+
+defineProps<{
+  longTexts: LongTexts
+}>()
+</script>
+
+<template>
+  <PreviewRow v-if="longTexts.tenor">
+    <PreviewCategory>Tenor</PreviewCategory>
+    <PreviewContent>
+      <TextEditor
+        id="previewTenor"
+        aria-label="Tenor"
+        field-size="max"
+        preview
+        :value="longTexts.tenor"
+      />
+    </PreviewContent>
+  </PreviewRow>
+  <PreviewRow v-if="longTexts.reasons">
+    <PreviewCategory>Gründe</PreviewCategory>
+    <PreviewContent>
+      <TextEditor
+        id="previewReasons"
+        aria-label="Gründe"
+        field-size="max"
+        preview
+        :value="longTexts.reasons"
+      />
+    </PreviewContent>
+  </PreviewRow>
+  <PreviewRow v-if="longTexts.caseFacts">
+    <PreviewCategory>Tatbestand</PreviewCategory>
+    <PreviewContent>
+      <TextEditor
+        id="previewCaseFacts"
+        aria-label="Tatbestand"
+        field-size="max"
+        preview
+        :value="longTexts.caseFacts"
+      />
+    </PreviewContent>
+  </PreviewRow>
+  <PreviewRow v-if="longTexts.decisionReasons">
+    <PreviewCategory>Entscheidungsgründe</PreviewCategory>
+    <PreviewContent>
+      <TextEditor
+        id="previewDecisionReasons"
+        aria-label="Entscheidungsgründe"
+        field-size="max"
+        preview
+        :value="longTexts.decisionReasons"
+      />
+    </PreviewContent>
+  </PreviewRow>
+  <PreviewRow v-if="longTexts.otherLongText">
+    <PreviewCategory>Sonstiger Langtext</PreviewCategory>
+    <PreviewContent>
+      <TextEditor
+        id="previewOtherLongText"
+        aria-label="Sonstiger Langtext"
+        field-size="max"
+        preview
+        :value="longTexts.otherLongText"
+      />
+    </PreviewContent>
+  </PreviewRow>
+  <PreviewRow v-if="longTexts.dissentingOpinion">
+    <PreviewCategory>Abweichende Meinung</PreviewCategory>
+    <PreviewContent>
+      <TextEditor
+        id="previewDissentingOpinion"
+        aria-label="Abweichende Meinung"
+        field-size="max"
+        preview
+        :value="longTexts.dissentingOpinion"
+      />
+    </PreviewContent>
+  </PreviewRow>
+  <PreviewRow v-if="longTexts.outline">
+    <PreviewCategory>Gliederung</PreviewCategory>
+    <PreviewContent>
+      <TextEditor
+        id="previewOutline"
+        aria-label="Gliederung"
+        field-size="max"
+        preview
+        :value="longTexts.outline"
+      />
+    </PreviewContent>
+  </PreviewRow>
+</template>
```

Comment:
This is a lot of duplicated code. Can we map over the data and render it dynamically like so for example?

```
defineProps<{
  longTexts: LongTexts
}>()

const longTextFields = {
  tenor: "Tenor",
  reasons: "Gründe",
  caseFacts: "Tatbestand",
  decisionReasons: "Entscheidungsgründe",
  otherLongText: "Sonstiger Langtext",
  dissentingOpinion: "Abweichende Meinung",
  outline: "Gliederung",
}
</script>

<template>
  <div v-for="(label, key) in longTextFields" :key="key">
    <PreviewRow v-if="longTexts[key]">
      <PreviewCategory>{{ label }}</PreviewCategory>
      <PreviewContent>
        <TextEditor
          :id="'preview' + key.charAt(0).toUpperCase() + key.slice(1)"
          :aria-label="label"
          field-size="max"
          preview
          :value="longTexts[key]"
        />
      </PreviewContent>
    </PreviewRow>
  </div>
</template>
```